doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,23 @@
 
 <h3>New features since last release</h3>
 
+* The function `qml.ctrl` was given the optional argument `control_values=None`.
```

Comment:
This would probably be considered a "Improvement" rather than a "new feature", since it's changing something that already exists in PennyLane.  We probably also don't need an example for something of this scope, since those interested can check the documentation.

Also a heads up that we are in the feature freeze for a new release, so this changelog file will be reset soon. That cause some merge conflicts when updating to the current master.

Reply:
Hi, thanks for the feedback! I moved it to improvements and removed the example. If any merge conflicts arise, I will happily sort them out.